package.json

```diff
@@ -9,13 +9,13 @@
   ],
   "scripts": {
     "lint": "aegir-lint",
-    "test": "gulp test",
+    "test": "gulp test --webworker false",
```

Comment:
They are failing, and since aegir 10 are enabled by default. Need to check why they are failing at another point